### DISCUSSION TITLE
Stata License Renewal

### DISCUSSION THREAD

**Discussion by pew36** [gui2de/ppol768-spring23#122] [General]
Does anyone know when we should expect to receive new authorization codes to renew our Stata licenses?

**Comment by BeaLeydier** [comment]
They should be updated tomorrow on On The Hub (where you downloaded Stata from). I will post it here once we have it as well, in case it doesn't get updated right away or some people don't find it.

**Comment by bbdaniels** [comment]
> The university Stata license has been updated. I simply updated the two fields mentioned below in the renewal instructions. I have alerted all MPP and MIDP students but thought I'd do the same for you. If you are running Stata from GU, please read the instructions before renewing your license.
> 
> Regards,
> 
> Eric
> 
> Renewal instructions
> 
> On February 7, 2023, the University’s Stata 17/BE license expired. The updated license codes push your expiration to 7 Feb 2024. I have already updated my license.
> 
> Please see instructions on how to update Stata below. You DO NOT need to re-download Stata. If you are an MPP-EP student running Stata 16 this process might still work. The new license information has been pushed to your Software Webstore account. All students should follow the instructions below. I've been assured by UIS the new license information will be in your previous order.
> 
>  The new license information should have been pushed to your previous Stata order. You should see: License Expiration Date: 7 February 2024. Here are the instructions:
> 
> > Log into the [Georgetown Software webstore](https://georgetown.onthehub.com/WebStore/Welcome.aspx) and click the down arrow (v) where it says “Hello, Your name” in the upper right of your screen and then go to Your Account/Orders.
> 
> > Find your previous Stata order and then click View Details. The new license information should be there with the 2024 expiration date. Note: YOU SHOULD NOT NEED TO RE-DOWNLOAD STATA. SIMPLY UPDATE THE LICENSE.
> 
> > In Stata, go to the Help menu and click on ‘About Stata’. Then, ‘Update License’.
> 
> >The serial # has not changed. Just copy/paste the new code and Authorization key into the dialog box. Your ‘flavor’ is Stata BE.
> 
> >Note: Stata 16 users only: UIS said the process is apparently the same for those using Stata 16. If it doesn’t work, just download Stata 17 (BE) from the software webstore. Maybe Stata 16 doesn’t need to be updated but I suspect it does.
> 
>  Feel free to reach out with questions.
